notebook/static/notebook/js/notebook.js

```diff
@@ -771,27 +750,23 @@ define(function (require) {
                 if (this.mode !== 'command') {
                     this.command_mode();
                 }
+                this.get_cell(sindex).unselect();
```

Comment:
Hmm, this worries me - the logic in select is super subtle. Why is this needed now?